recipes/reef3d/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "reef3d" %}
+{% set version = "23.08" %}
+{% set sha256 = "9e114697b032615b16a4ce1459b91474944caf3b3ac182a0ddd5d733681c4549" %}
+
+{% set mpi = mpi or 'openmpi' %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/REEF3D/REEF3D/archive/refs/tags/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - make
+    - {{ compiler('cxx') }}
+  host:
+    - eigen =3.3.8
+    - hypre
+    - {{ mpi }}
+  run:
+    - hypre
+    - {{ mpi }}
+    - divemesh
+    - libcxx
+
+test:
+  commands:
+    - test -f $PREFIX/bin/reef3d
+    - reef3d  # [osx]
```

Comment:
```suggestion
# Only test on osx because of missing mpi ORTE
    - reef3d  # [osx]
```
Please add explanatory notes for strange selectors, so reviewers don't have to ask / review commits for the whys. The git history of this recipe will be not be available in the feedstock.

Reply:
Thank you @carterbox. FWIW, I agree this is a useful comment, but I was trying to follow the FAQ to the letter, which says to remove all comments: https://github.com/conda-forge/staged-recipes/blob/main/README.md#faq

Potentially an opportunity for updating the FAQ?

Thanks again for helping me get this merged!

Reply:
Yes! The intention was to have you remove all the comments from the example template.